### DISCUSSION TITLE
[Idea] Caption on pip

### DISCUSSION THREAD

**Discussion by Rayazayn** [PrestaShop/PrestaShop#36226] [Ideas]
### The problem

It's really gets frustrated when I am on pip and doing something while still watching the video, but there is no caption on pips so it's becomes really irritating when I am watching something with a language that don't know 

### The solution

If there is anyway making subtitles available on pips that would be great

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
Hello @Rayazayn I'm confused, are you sure you did not want to open this to https://github.com/pypa/pip ?

**Reply by Progi1984** [reply]
I think it's Picture in Picture : https://en.wikipedia.org/wiki/Picture-in-picture

**Comment by Rayazayn** [comment]
Yes I was talking about picture in picture

**Reply by kpodemski** [reply]
Hello @Rayazayn 

I think you didn't mean to post a discussion here on PrestaShop Github Discussions. We don't provide any video playback script or any other similar feature 🤔

**Comment by matks** [comment]
I close this discussion then ;)

@Rayazayn we cannot help you you need to go contact people of picture in picture
